src/views/Approvals/index.tsx

```diff
@@ -0,0 +1,167 @@
+        } else if (focusedCard > data[n].items.length - 1) {
+          setFocusedCard(Math.max(0, data[n].items.length - 1))
+        }
+
+        // Don't let view navigation handle this
+        event.stopPropagation()
+      } else {
+        const n = focusedColumn || 0
+        const l = data[n].items.length
+        const m = (focusedCard === undefined)
+          ? (event.key === 'ArrowDown' ? 0 : l - 1)
+          : (focusedCard + (event.key === 'ArrowDown' ? 1 : -1) + l) % l
+
+        if (m !== focusedColumn) {
+          setFocusedColumn(n)
+        }
+        setFocusedCard(m)
+      }
+    }
+  })
+
+  return (
+    <View.Root>
+      <ViewHeader.Root>
+        <ViewHeader.Title title='Dagen' short='Dagen' iconColor='#5E9F5D' icon={EarthIcon} />
+      </ViewHeader.Root>
+
+      <View.Content variant='grid' columns={slots.length}>
+        {data.map((slot, colN) => {
+          return (
+            <View.Column key={slot.key}>
+              <TimeSlot label={slot.label || ''} slots={slot.hours || []} />
+
+              {slot.items.map((assignment, cardN) => {
+                const articleId = assignment.links.find((l) => l.type === 'core/article')?.uuid
+                const time = assignment.data.publish
+                  ? format(toZonedTime(parseISO(assignment.data.publish), timeZone), 'HH:mm')
+                  : undefined
+
+                const isSelected = ((articleId && openEditors.includes(articleId)) || openPlannings.includes(assignment._id))
+                return (
+                  <Card.Root
+                    key={assignment.id}
+                    className={(!articleId) ? 'opacity-50' : 'hover:bg-muted'}
+                    isFocused={colN === focusedColumn && cardN === focusedCard}
+                    isSelected={isSelected}
+                    onSelect={(event) => {
+                      if (event instanceof KeyboardEvent && event.key == ' ') {
+                        openPlanning(event, { id: assignment._id })
+                      } else if (articleId) {
+                        openArticle(event, { id: articleId })
+                      }
+                    }}
+                  >
+                    <Card.Header>
+                      <div>{assignment._newsvalue}</div>
+                      <div className='flex flex-row gap-1 items-center'>
+                        <ClockIcon hour={(time) ? parseInt(time.slice(0, 2)) : undefined} size={14} className='opacity-50' />
+                        <time>{time}</time>
+                      </div>
+                    </Card.Header>
+
+                    <Card.Content>
+                      <Card.Title>{assignment.title}</Card.Title>
+                      <Card.Body>{assignment.meta.find((m) => m.type === 'tt/slugline')?.value || '-'}</Card.Body>
+                    </Card.Content>
+
+                    <Card.Footer>
+                      {assignment._section}
+                      &middot;
+                      Anders Andersson/TT
```

Comment:
Should we not type out the assignee, if found, here instead of a placeholder?

Reply:
Definitely! Forgot to fix that. I'll get to it in follow up issues.